### ISSUE TITLE
OoT64 decomp: unofficial "contributor agreement"

### ISSUE THREAD

**Issue by Dragorn421** [Dragorn421/oot-licensing#3]
Hi!

You are mentioned here because you have submitted a contribution to the [zeldaret/oot repo](https://github.com/zeldaret/oot), a decompilation of the game Ocarina of Time 64.

I would like to have you release that contribution of yours and all future contributions to the oot decomp under [CC0](https://creativecommons.org/publicdomain/zero/1.0/) ("public domain").

- If you don't care, the best choice to do for the project and users of the project (researchers, fans, modders...) is to agree.
- If you agree, you can reply to this issue with some message like "I agree to share all my contributions to the oot decomp under CC0" (please include MM in the statement if you also contribute to the MM decomp or plan to)
- If you disagree, please get in touch! Through [github issues](https://github.com/Dragorn421/oot-licensing/issues) or [discord](https://discord.gg/rDvMsgnU2m)

---------

(this is much like https://github.com/Dragorn421/oot-licensing/issues/1 but for future contributors instead of past contributors)

The repo isn't fully covered by CC0 yet (see https://github.com/Dragorn421/oot-licensing/issues/1 and the [README](https://github.com/Dragorn421/oot-licensing/blob/main/README.md) for details) but it is a goal that it is.

<details>

<summary>Archived</summary>

https://web.archive.org/web/20230114011343/https://github.com/Dragorn421/oot-licensing/issues/3

</details>

**Comment by Dragorn421** [comment]
Hello @waffleoRai !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1236
See the top comment of this issue to see what this mention is about

**Comment by waffleoRai** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by Dragorn421** [comment]
Hello @Hyena-Senpai !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1500
See the top comment of this issue to see what this mention is about

**Comment by Dragorn421** [comment]
Hello @adaliaramon !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1508
See the top comment of this issue to see what this mention is about

**Comment by adaliaramon** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Dragorn421** [comment]
Hello @hack-nuss

You will eventually be contributing to the oot decomp repo through zeldaret/oot#1548
See the top comment of this issue to see what this mention is about

**Comment by hack-nuss** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Dragorn421** [comment]
Hello @cadmic

You will eventually be contributing to the oot decomp repo through zeldaret/oot#1557
See the top comment of this issue to see what this mention is about

**Comment by cadmic** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Dragorn421** [comment]
Hello @javarooster-png

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1576
See the top comment of this issue to see what this mention is about

**Comment by Javarooster-png** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by Dragorn421** [comment]
Hello @hiisuya

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1581
See the top comment of this issue to see what this mention is about

**Comment by hiisuya** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by Dragorn421** [comment]
Hello @adaliaramon 

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1602
See the top comment of this issue to see what this mention is about

**Comment by Dragorn421** [comment]
Hello @phil-macrocheira

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/1603
See the top comment of this issue to see what this mention is about

**Comment by phil-macrocheira** [comment]
I agree to share all my contributions to the oot and mm decomp under CC0

**Comment by inspectredc** [comment]
I agree to share all my contributions to the oot and mm decomp under CC0

**Comment by Pepe20129** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by Dragorn421** [comment]
Hello @Ozero4

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2124
See the top comment of this issue to see what this mention is about

**Comment by Ozero4** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by Dragorn421** [comment]
Hello @Feacur

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2283
See the top comment of this issue to see what this mention is about

**Comment by Feacur** [comment]
https://github.com/Dragorn421/oot-licensing/issues/3#issuecomment-2454600995

oh, hi there! sure thing, i do agree to share all my contributions to the oot and mm decomps under CC0.

not sure if i plan to do so with mm right now, but i might and agree in advance.

**Comment by Dragorn421** [comment]
Hello @JordanLongstaff

You will eventually be contributing to the oot decomp repo through zeldaret/oot#2288
See the top comment of this issue to see what this mention is about

**Comment by JordanLongstaff** [comment]
I agree to share all my contributions to the oot decomp (and any other decomp projects including MM) under CC0.

**Comment by benvillalobos** [comment]
I agree to share all my contributions to the oot / mm decomp under CC0

**Comment by GabrielRavier** [comment]
I agree to share all my contributions to the OOT and MM decomps under CC0

**Comment by Dragorn421** [comment]
Hello @hamiltop

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2682
See the top comment of this issue to see what this mention is about

**Comment by Dragorn421** [comment]
Hello @serprex !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2730
See the top comment of this issue to see what this mention is about

**Comment by serprex** [comment]
I agree to share all my contributions to the OoT or MM decomp under CC0

**Comment by Dragorn421** [comment]
Hello @scudo005 !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2745
See the top comment of this issue to see what this mention is about

**Comment by Dragorn421** [comment]
Hello @nathan-casabieille !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2747
See the top comment of this issue to see what this mention is about

**Comment by nathan-casabieille** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by scudo005** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Dragorn421** [comment]
Hello @Olvivi !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2807
See the top comment of this issue to see what this mention is about

**Comment by Olvivi** [comment]
I agree to share my contributions under CC0.

**Comment by Dragorn421** [comment]
Hello @chasedream1129 !

You will eventually be contributing to the oot decomp repo through https://github.com/zeldaret/oot/pull/2827
See the top comment of this issue to see what this mention is about

**Comment by chasedream1129** [comment]
I agree to share all my contributions to the oot decomp under CC0
